charts/dynamic-link-server/files/README.md

```diff
@@ -0,0 +1,28 @@
+# Dynamic links server
+
+This mock server is designed to test crawling on a target with links, which appear on the 'load' event, after a small delay and on different user interactions.
```

Comment:
This file isn't required. But you can take this as a description for the Helm chart

Reply:
Done